<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite name="CodeIgniter Application Test Suite" tests="22" assertions="23" errors="0" failures="0" skipped="2" time="0.309540">
    <testsuite name="gerenciar_rodoviaria_controller_test" file="C:\Users\sandr\xampp\htdocs\rpv\application\tests\controllers\gerenciar_rodoviaria_controller_test.php" tests="3" assertions="3" errors="0" failures="0" skipped="0" time="0.001157">
      <testcase name="testIndex" class="gerenciar_rodoviaria_controller_test" classname="gerenciar_rodoviaria_controller_test" file="C:\Users\sandr\xampp\htdocs\rpv\application\tests\controllers\gerenciar_rodoviaria_controller_test.php" line="13" assertions="1" time="0.000510"/>
      <testcase name="testErrorMessenge" class="gerenciar_rodoviaria_controller_test" classname="gerenciar_rodoviaria_controller_test" file="C:\Users\sandr\xampp\htdocs\rpv\application\tests\controllers\gerenciar_rodoviaria_controller_test.php" line="18" assertions="1" time="0.000312"/>
      <testcase name="testSucessMessenge" class="gerenciar_rodoviaria_controller_test" classname="gerenciar_rodoviaria_controller_test" file="C:\Users\sandr\xampp\htdocs\rpv\application\tests\controllers\gerenciar_rodoviaria_controller_test.php" line="22" assertions="1" time="0.000335"/>
    </testsuite>
    <testsuite name="Cidade_model_test" file="C:\Users\sandr\xampp\htdocs\rpv\application\tests\models\Cidade_model_test.php" tests="4" assertions="4" errors="0" failures="0" skipped="0" time="0.077507">
      <testcase name="testGetCidadeError" class="Cidade_model_test" classname="Cidade_model_test" file="C:\Users\sandr\xampp\htdocs\rpv\application\tests\models\Cidade_model_test.php" line="20" assertions="1" time="0.013591">
        <system-out>&#13;
&#13;
</system-out>
      </testcase>
      <testcase name="testGetCidadeSucess" class="Cidade_model_test" classname="Cidade_model_test" file="C:\Users\sandr\xampp\htdocs\rpv\application\tests\models\Cidade_model_test.php" line="26" assertions="1" time="0.021136">
        <system-out>&#13;
&#13;
&#13;
A PHP Error was encountered&#13;
&#13;
Severity:    Notice
Message:     Undefined index: cidade_nome
Filename:    C:\Users\sandr\xampp\htdocs\rpv\application\tests\models\Cidade_model_test.php
Line Number: 29&#13;
&#13;
Backtrace:&#13;
	File: C:\Users\sandr\xampp\htdocs\rpv\application\tests\models\Cidade_model_test.php
	Line: 29
	Function: _error_handler

	File: phar://C:/Users/sandr/xampp/php/phpunit/phpunit/Framework/TestCase.php
	Line: 1153
	Function: testGetCidadeSucess

	File: phar://C:/Users/sandr/xampp/php/phpunit/phpunit/Framework/TestCase.php
	Line: 842
	Function: runTest

	File: phar://C:/Users/sandr/xampp/php/phpunit/phpunit/Framework/TestResult.php
	Line: 687
	Function: runBare

	File: phar://C:/Users/sandr/xampp/php/phpunit/phpunit/Framework/TestCase.php
	Line: 796
	Function: run

	File: phar://C:/Users/sandr/xampp/php/phpunit/phpunit/Framework/TestSuite.php
	Line: 746
	Function: run

	File: phar://C:/Users/sandr/xampp/php/phpunit/phpunit/Framework/TestSuite.php
	Line: 746
	Function: run

	File: phar://C:/Users/sandr/xampp/php/phpunit/phpunit/TextUI/TestRunner.php
	Line: 641
	Function: run

	File: phar://C:/Users/sandr/xampp/php/phpunit/phpunit/TextUI/Command.php
	Line: 206
	Function: doRun

	File: phar://C:/Users/sandr/xampp/php/phpunit/phpunit/TextUI/Command.php
	Line: 162
	Function: run

	File: C:\Users\sandr\xampp\php\phpunit
	Line: 614
	Function: main

&#13;
</system-out>
      </testcase>
      <testcase name="testGetCidadeEspecifica" class="Cidade_model_test" classname="Cidade_model_test" file="C:\Users\sandr\xampp\htdocs\rpv\application\tests\models\Cidade_model_test.php" line="32" assertions="1" time="0.020011">
        <system-out>&#13;
&#13;
</system-out>
      </testcase>
      <testcase name="testGetTodasCidades" class="Cidade_model_test" classname="Cidade_model_test" file="C:\Users\sandr\xampp\htdocs\rpv\application\tests\models\Cidade_model_test.php" line="38" assertions="1" time="0.022770">
        <system-out>&#13;
&#13;
</system-out>
      </testcase>
    </testsuite>
    <testsuite name="Estado_model_test" file="C:\Users\sandr\xampp\htdocs\rpv\application\tests\models\Estado_model_test.php" tests="6" assertions="6" errors="0" failures="0" skipped="0" time="0.093484">
      <testcase name="testGetEstadoError" class="Estado_model_test" classname="Estado_model_test" file="C:\Users\sandr\xampp\htdocs\rpv\application\tests\models\Estado_model_test.php" line="19" assertions="1" time="0.017935">
        <system-out>&#13;
&#13;
</system-out>
      </testcase>
      <testcase name="testGetEstadoSucess" class="Estado_model_test" classname="Estado_model_test" file="C:\Users\sandr\xampp\htdocs\rpv\application\tests\models\Estado_model_test.php" line="25" assertions="1" time="0.022475">
        <system-out>&#13;
&#13;
&#13;
A PHP Error was encountered&#13;
&#13;
Severity:    Notice
Message:     Undefined index: estado_nome
Filename:    C:\Users\sandr\xampp\htdocs\rpv\application\tests\models\Estado_model_test.php
Line Number: 28&#13;
&#13;
Backtrace:&#13;
	File: C:\Users\sandr\xampp\htdocs\rpv\application\tests\models\Estado_model_test.php
	Line: 28
	Function: _error_handler

	File: phar://C:/Users/sandr/xampp/php/phpunit/phpunit/Framework/TestCase.php
	Line: 1153
	Function: testGetEstadoSucess

	File: phar://C:/Users/sandr/xampp/php/phpunit/phpunit/Framework/TestCase.php
	Line: 842
	Function: runTest

	File: phar://C:/Users/sandr/xampp/php/phpunit/phpunit/Framework/TestResult.php
	Line: 687
	Function: runBare

	File: phar://C:/Users/sandr/xampp/php/phpunit/phpunit/Framework/TestCase.php
	Line: 796
	Function: run

	File: phar://C:/Users/sandr/xampp/php/phpunit/phpunit/Framework/TestSuite.php
	Line: 746
	Function: run

	File: phar://C:/Users/sandr/xampp/php/phpunit/phpunit/Framework/TestSuite.php
	Line: 746
	Function: run

	File: phar://C:/Users/sandr/xampp/php/phpunit/phpunit/TextUI/TestRunner.php
	Line: 641
	Function: run

	File: phar://C:/Users/sandr/xampp/php/phpunit/phpunit/TextUI/Command.php
	Line: 206
	Function: doRun

	File: phar://C:/Users/sandr/xampp/php/phpunit/phpunit/TextUI/Command.php
	Line: 162
	Function: run

	File: C:\Users\sandr\xampp\php\phpunit
	Line: 614
	Function: main

&#13;
</system-out>
      </testcase>
      <testcase name="testGetUFSucess" class="Estado_model_test" classname="Estado_model_test" file="C:\Users\sandr\xampp\htdocs\rpv\application\tests\models\Estado_model_test.php" line="31" assertions="1" time="0.019295">
        <system-out>&#13;
&#13;
&#13;
A PHP Error was encountered&#13;
&#13;
Severity:    Notice
Message:     Undefined index: estado_uf
Filename:    C:\Users\sandr\xampp\htdocs\rpv\application\tests\models\Estado_model_test.php
Line Number: 34&#13;
&#13;
Backtrace:&#13;
	File: C:\Users\sandr\xampp\htdocs\rpv\application\tests\models\Estado_model_test.php
	Line: 34
	Function: _error_handler

	File: phar://C:/Users/sandr/xampp/php/phpunit/phpunit/Framework/TestCase.php
	Line: 1153
	Function: testGetUFSucess

	File: phar://C:/Users/sandr/xampp/php/phpunit/phpunit/Framework/TestCase.php
	Line: 842
	Function: runTest

	File: phar://C:/Users/sandr/xampp/php/phpunit/phpunit/Framework/TestResult.php
	Line: 687
	Function: runBare

	File: phar://C:/Users/sandr/xampp/php/phpunit/phpunit/Framework/TestCase.php
	Line: 796
	Function: run

	File: phar://C:/Users/sandr/xampp/php/phpunit/phpunit/Framework/TestSuite.php
	Line: 746
	Function: run

	File: phar://C:/Users/sandr/xampp/php/phpunit/phpunit/Framework/TestSuite.php
	Line: 746
	Function: run

	File: phar://C:/Users/sandr/xampp/php/phpunit/phpunit/TextUI/TestRunner.php
	Line: 641
	Function: run

	File: phar://C:/Users/sandr/xampp/php/phpunit/phpunit/TextUI/Command.php
	Line: 206
	Function: doRun

	File: phar://C:/Users/sandr/xampp/php/phpunit/phpunit/TextUI/Command.php
	Line: 162
	Function: run

	File: C:\Users\sandr\xampp\php\phpunit
	Line: 614
	Function: main

&#13;
</system-out>
      </testcase>
      <testcase name="testGetEstadoEspecifico" class="Estado_model_test" classname="Estado_model_test" file="C:\Users\sandr\xampp\htdocs\rpv\application\tests\models\Estado_model_test.php" line="37" assertions="1" time="0.017117">
        <system-out>&#13;
&#13;
</system-out>
      </testcase>
      <testcase name="testGetUFEspecifico" class="Estado_model_test" classname="Estado_model_test" file="C:\Users\sandr\xampp\htdocs\rpv\application\tests\models\Estado_model_test.php" line="42" assertions="1" time="0.007679">
        <system-out>&#13;
&#13;
</system-out>
      </testcase>
      <testcase name="testGetTodosEstados" class="Estado_model_test" classname="Estado_model_test" file="C:\Users\sandr\xampp\htdocs\rpv\application\tests\models\Estado_model_test.php" line="49" assertions="1" time="0.008983">
        <system-out>&#13;
&#13;
</system-out>
      </testcase>
    </testsuite>
    <testsuite name="Rodoviaria_model_test" file="C:\Users\sandr\xampp\htdocs\rpv\application\tests\models\Rodoviaria_model_test.php" tests="9" assertions="10" errors="0" failures="0" skipped="2" time="0.137392">
      <testcase name="testGetRodoviariasCidadeTest" class="Rodoviaria_model_test" classname="Rodoviaria_model_test" file="C:\Users\sandr\xampp\htdocs\rpv\application\tests\models\Rodoviaria_model_test.php" line="22" assertions="4" time="0.023257">
        <system-out>&#13;
&#13;
</system-out>
      </testcase>
      <testcase name="testGetRodoviariaeEspecifica" class="Rodoviaria_model_test" classname="Rodoviaria_model_test" file="C:\Users\sandr\xampp\htdocs\rpv\application\tests\models\Rodoviaria_model_test.php" line="34" assertions="1" time="0.013618">
        <system-out>&#13;
&#13;
</system-out>
      </testcase>
      <testcase name="testInsertionRodoviariaCidadeCorreta" class="Rodoviaria_model_test" classname="Rodoviaria_model_test" file="C:\Users\sandr\xampp\htdocs\rpv\application\tests\models\Rodoviaria_model_test.php" line="43" assertions="1" time="0.018411">
        <system-out>&#13;
&#13;
</system-out>
      </testcase>
      <testcase name="testInsertionRodoviariaCidadeIncorreta" class="Rodoviaria_model_test" classname="Rodoviaria_model_test" file="C:\Users\sandr\xampp\htdocs\rpv\application\tests\models\Rodoviaria_model_test.php" line="52" assertions="1" time="0.018839">
        <system-out>&#13;
&#13;
</system-out>
      </testcase>
      <testcase name="testOverflowInsertionRodoviaria" class="Rodoviaria_model_test" classname="Rodoviaria_model_test" file="C:\Users\sandr\xampp\htdocs\rpv\application\tests\models\Rodoviaria_model_test.php" line="64" assertions="0" time="0.009753">
        <skipped/>
        <system-out>&#13;
&#13;
</system-out>
      </testcase>
      <testcase name="testUnexpectedTypeInsertionRodoviaria" class="Rodoviaria_model_test" classname="Rodoviaria_model_test" file="C:\Users\sandr\xampp\htdocs\rpv\application\tests\models\Rodoviaria_model_test.php" line="81" assertions="0" time="0.010943">
        <skipped/>
        <system-out>&#13;
&#13;
</system-out>
      </testcase>
      <testcase name="testGetQuantRodoviariaPorCidade" class="Rodoviaria_model_test" classname="Rodoviaria_model_test" file="C:\Users\sandr\xampp\htdocs\rpv\application\tests\models\Rodoviaria_model_test.php" line="91" assertions="1" time="0.009739">
        <system-out>&#13;
&#13;
</system-out>
      </testcase>
      <testcase name="testUpdateRodoviariaSuccess" class="Rodoviaria_model_test" classname="Rodoviaria_model_test" file="C:\Users\sandr\xampp\htdocs\rpv\application\tests\models\Rodoviaria_model_test.php" line="99" assertions="1" time="0.018279">
        <system-out>&#13;
&#13;
</system-out>
      </testcase>
      <testcase name="testUpdateRodoviariaError" class="Rodoviaria_model_test" classname="Rodoviaria_model_test" file="C:\Users\sandr\xampp\htdocs\rpv\application\tests\models\Rodoviaria_model_test.php" line="105" assertions="1" time="0.014553">
        <system-out>&#13;
&#13;
</system-out>
      </testcase>
    </testsuite>
  </testsuite>
</testsuites>
